example/basic_widget.html

```diff
@@ -219,7 +219,9 @@
         // --------------------- Setup Guardian JS ----------------------------
 
         var guardianjs = auth0GuardianJS({
-          serviceUrl: "https://{{ userData.tenant }}.guardian.auth0.com", // https://{name}.guardian.auth0.com
+          // e.g: "https://{{ userData.tenant }}.eu.guardian.auth0.com" for Europe  
```

Comment:
I guess if we are using `mfaServerUrl` we don't need the comments anymore.

Reply:
That's a good point, though on having the comment so you know what it is supposed to point to, but actually it is better to handle it as opaque so will remove